### ISSUE TITLE
conan-center hook always reports "pre_export(): could not get source code" on WIndows

### ISSUE THREAD

**Issue by tt4g** [conan-io/hooks#157]
I tried to build after enabling conan-center hook on Windows 10 but I get error message "ERROR: [HOOK - conan-center.py] pre_export(): could not get source code".
This recipe can build on docker container conanio/gcc7.

Is conan-center hook not supported Windows 10?

**Comment by tt4g** [comment]
I'm using conan installed by Windows installer, that is bad?

**Comment by uilianries** [comment]
Hi @tt4g which Conan client version are you running? Where did you install conan hook files (system path e.g. C:/Users/foobar/.conan)?

**Comment by tt4g** [comment]
I'm using 1.22.2:

```bash
$ conan --version
Conan version 1.22.2
```

Hooks installed here: `C:\Users\tt\.conan\hooks`, and list:

```bash
$ ls -lha
total 89K
drwxr-xr-x 1 tt 197611    0  2月 16 14:55 ./
drwxr-xr-x 1 tt 197611    0  2月 16 15:50 ../
-rw-r--r-- 1 tt 197611  421  2月 16 15:50 attribute_checker.py
-rwxr-xr-x 1 tt 197611 9.1K  2月 16 15:50 binary_linter.py*
-rw-r--r-- 1 tt 197611  11K  2月 16 15:50 bintray_updater.py
-rw-r--r-- 1 tt 197611  31K  2月 16 15:50 conan-center.py
-rw-r--r-- 1 tt 197611 3.2K  2月 16 15:50 export_metadata.py
-rwxr-xr-x 1 tt 197611 6.6K  2月 16 15:50 github_updater.py*
-rw-r--r-- 1 tt 197611 1.2K  2月 16 15:50 members_typo_checker.py
-rw-r--r-- 1 tt 197611 3.4K  2月 16 15:50 recipe_linter.py
-rw-r--r-- 1 tt 197611  920  2月 16 15:50 spdx_checker.py
```
conan Installed: `C:\Program Files\Conan`
conan  executable file: `C:\Program Files\Conan\conan\conan.exe`

**Comment by tt4g** [comment]
I challenged by disabling the Windows application alias. However, I got the same message as the issue.

* [Frequently Asked Questions about using Python on Windows \| Microsoft Docs](https://docs.microsoft.com/en-us/windows/python/faqs)
* [Typing “python” on Windows 10 \(version 1903\) command prompt opens Microsoft store \- Super User](https://superuser.com/questions/1437590/typing-python-on-windows-10-version-1903-command-prompt-opens-microsoft-stor)

**Comment by memsharded** [comment]
Yes, the Windows installers bundles a python interpreter in it, so it might be an issue. If you could try to run in Windows with the pip-installed Conan, that would help to isolate the issue.

**Comment by tt4g** [comment]
@memsharded Right. conan installed by pip is works for me.

I was uninstalling python because Windows application alias often do bad when calling python3. That was bad.

**Comment by memsharded** [comment]
Thanks @tt4g for following up!

**Comment by uilianries** [comment]
@memsharded should we investigate Windows installer?

**Comment by memsharded** [comment]
@uilianries yes, better. Please do a quick check if the hooks work for the Windows installer. Thanks!

**Comment by steinerthomas** [comment]
The hooks still doesn't work on for the Windows installer. I also had to install conan via pip.
@uilianries any updates?

**Comment by claremacrae** [comment]
The same error occurs when running conan installed via [choco](https://chocolatey.org/packages/conan).

**Comment by alejandro-colomar** [comment]
I got this problem using the debian installer on Debian Sid

**Comment by uilianries** [comment]
I think this problem deserves further investigation, but one issue per package manager/platform. I'll check for Debian first

**Comment by AGreat1** [comment]
TLDR: make sure the hooks in your config still exist.

Here's my two cents: I have conan installed with the windows installer. I had upgraded versions, and as someone who has been contributing to the conan-center once in a while, I had some hooks. I had been getting the "could not get source code" error. I eventually looked into my hook config to take them out. I noticed I had the (old?) attribute-checker and the conan-center hooks. Out of curiosity, I removed only the attribute checker, and now it works...

I don't know whether this is my mistake handling hooks, or that I was supposed to uninstall conan before installing the new one, or it was a mistake in the `conan config install`.

**Comment by chausner** [comment]
I am running into this issue on Windows. I installed Conan via winget.

**Comment by HappySeaFox** [comment]
I also face the issue both on Windows 7 x64 and LUbuntu 21.04. I installed conan from the Downloads page on both the systems.
